.travis.yml

```diff
@@ -17,8 +17,7 @@ env:
     # test single node/rank with different threading back ends
     - BUILD_NAME=serial   WITH_THREAD=serial   WITH_DISTRIBUTED=serial
     - BUILD_NAME=cthread  WITH_THREAD=cthread  WITH_DISTRIBUTED=serial
-    # The tbb backend sometimes fails to build because CMake. Turn it off until our CMake TBB support is stable.
-    #- BUILD_NAME=tbb      WITH_THREAD=tbb      WITH_DISTRIBUTED=serial
+    - BUILD_NAME=tbb      WITH_THREAD=tbb      WITH_DISTRIBUTED=serial
     # test mpi
     - BUILD_NAME=mpi      WITH_THREAD=cthread  WITH_DISTRIBUTED=mpi
```

Comment:
Should we test mpi also with tbb?

Reply:
Positive:
* more test coverage

Neutral:
* we already test the combination of threads + mpi (with threads provided by cthread)

Negative:
* tests take longer to run.

Your call.

Reply:
After looking inside of TBB, I can well imagine weird interactions and breakages --- it's not quite as professional as I'd imagined. I'd go for longer time here.

Reply:
I have added it.